go/adbc/driver/snowflake/driver_test.go

```diff
@@ -264,6 +264,7 @@ func createTempSchema(database string, uri string) string {
 }
 
 func dropTempSchema(uri, schema string) {
+
```

Comment:
accident? any particular reason to add the extra line?

Reply:
long story short, yes